lib/resque_bus/server.rb

```diff
@@ -22,6 +22,123 @@ def self.included(base)
         
       }
     end
+
+    class Helpers
+      class << self
+        def parse_query(query_string)
+          has_open_brace = query_string.include?("{")
```

Comment:
I know it's generally a flat JSON, but perhaps we could use a regex here to anchor the opening and closing brace to the start and end? I think we could also strip the string and then look at the first and last character.